### PR TITLE
Display course fees

### PR DESCRIPTION
**Pull request by dillonhammer**
## What
Display course fees

## How
Add course fee info to graphql declaration.

## Prereqs
This builds off of https://github.com/sandboxnu/searchneu/pull/83 and should be merged after that PR.

### CODE REVIEW

**Review by megandouli — APPROVED** [review]
lol it's confusing that the convert to graphql changes are here too

File: components/ResultsPage/Results/SearchResult.tsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ export function SearchResult({ course }: SearchResultProps): ReactElement {
 
   const feeString =
     course.feeDescription && course.feeAmount
-      ? `${course.feeDescription}- $${course.feeAmount}`
+      ? `${course.feeDescription} - $${course.feeAmount.toLocaleString()}`
```
Comment (NEUDitao) [inline components/ResultsPage/Results/SearchResult.tsx:64]: wait why is it `toLocaleString()`?

Comment (dillonhammer) [reply components/ResultsPage/Results/SearchResult.tsx:64]: The `feeAmount` is a number, that is to display it with commas every 3 digits. There are course fees in the thousands.

Comment (NEUDitao) [reply components/ResultsPage/Results/SearchResult.tsx:64]: javascript is spooky

**Review by NEUDitao — APPROVED** [review]

**Pull request merged**